fed/mosaic-application/src/main/java/org/eclipse/mosaic/fed/application/ambassador/simulation/communication/CellModule.java

```diff
@@ -66,17 +66,27 @@ public void enable(CellModuleConfiguration configuration) {
             owner.sendInteractionToRti(
                     new CellularCommunicationConfiguration(
                             owner.getSimulationTime(), new CellConfiguration(
-                            owner.getId(), true, configuration.getMaxDlBitrate(), configuration.getMaxUlBitrate())));
+                            owner.getId(), true, configuration.getMaxDownlinkBitrate(), configuration.getMaxUplinkBitrate())));
         } else {
             logEnableConfigurationNull();
         }
     }
 
     /**
-     * Convenience method to enable the cell module with default values.
+     * Convenience method to enable the cell module with bare minimum default values.
+     * Note: When using this method, no bitrates will be set. This could cause issues.
```

Comment:
what issues? am I still able to send CAMs then?

Reply:
Seems like I got lazy here. CAM sending wouldn't be possible if the `CellModule` is setup using this method.
I think users that want to use CAMs should configure their specify, that they want to use CAMs using either a a specialized configuration or the new method `enableWithCamDefaults()`.
Anyhow, this needs to be documented better.

Reply:
I additionally added a `null` check for the `camConfiguration` in the `sendCam()`-method in the `CellModule`

Reply:
Needs to be resolved with @rprotzmann